scraper.rb

```diff
@@ -40,7 +40,7 @@ class MemberPage < Scraped::HTML
   end
 
   field :image do
-    box.css('img/@src').first.text
+    box.css('img/@src').first.text rescue nil
```

Comment:
It's generally bad form to use exceptions for things that aren't actually exceptional. We abuse this from time to time, particularly in old-style scrapers where it makes the flow a lot easier. But one of the benefits of giving every field its own method is that we get to write things like this properly.